doc/plugin.txt

```diff
@@ -4,20 +4,23 @@ Synopsis
 
 Manage project plugins
 
-    add <plugin-spec> [...] ............ Add specified plugins
+    add <plugin-spec> [...] ............ Add specified plugins. If adding a plugin from npm, the
```

Comment:
I was concerned about the visibility of it, which is why I put it in the help. That being said, it did end up being longer than I intended so I am cool with moving it to the reference. I'll close this PR and open another.